web-app/css/openboxes.css

```diff
@@ -2873,6 +2881,79 @@ span.canceled, div.canceled {
     background-image: linear-gradient(#b33630, #dc5f59);
 }
 
+.button.helpscout {
+    /* base color is "PIH Teal" (#3AB4B1), with adjustments in HSL */
+
+    /* button gradient runs +/- 5% L from the base color */
+    background-color: #3AB4B1;
+    background-image: -webkit-gradient(linear, 0 0, 0 100%, from(#46C3C0), to(#34A29F));
+    background-image: -moz-linear-gradient(#46C3C0, #34A29F);
+    background-image: -ms-linear-gradient(#46C3C0, #34A29F);
+    background-image: -o-linear-gradient(#46C3C0, #34A29F);
+    background-image: linear-gradient(#46C3C0, #34A29F);
+
+    /* border colors are 10/15% darker */
+    border-color: #2E8F8C;
+    border-bottom-color: #287B79;
+    border-right-color: #287B79;
+
+    box-sizing: border-box;
+    color: #F3FBFB; /* 50% lighter than base color */
+    height: 30px;
+    text-decoration: none;
+    text-shadow: 1px 1px 0 #287B79; /* same as border-bottom-color */
+}
+
+.button.helpscout i {
+    filter: drop-shadow(1px 1px 0 #287B79); /* apply button's text-shadow to fontawesome */
+}
+
+.button.helpscout:focus,
+.button.helpscout:hover {
+    /* make button 10% darker on mouseover */
+    background-image: -webkit-gradient(linear, 0 0, 0 100%, from(#34A29F), to(#287B79));
+    background-image: -moz-linear-gradient(#34A29F, #287B79);
+    background-image: -ms-linear-gradient(#34A29F, #287B79);
+    background-image: -o-linear-gradient(#34A29F, #287B79);
+    background-image: linear-gradient(#34A29F, #287B79);
+    border-color: #287B79;
+    border-bottom-color: #226866;
+    border-right-color: #226866;
+    color: #E0F5F5;
+    text-shadow: 1px 1px 0 #226866; /* same as border-bottom-color */
+}
+
+.button.helpscout:focus i,
+.button.helpscout:hover i {
+    filter: drop-shadow(1px 1px 0 #226866); /* apply button's text-shadow to fontawesome */
+}
+
+.button.helpscout:active,
+.button.helpscout.active {
+    /* invert and darken button when pressed */
+    background-image: -webkit-gradient(linear, 0 0, 0 100%, from(#287B79), to(#34A29F));
+    background-image: -moz-linear-gradient(#287B79, #34A29F);
+    background-image: -ms-linear-gradient(#287B79, #34A29F);
+    background-image: -o-linear-gradient(#287B79, #34A29F);
+    background-image: linear-gradient(#287B79, #34A29F);
+    border-color: #287B79;
+    border-top-color: #226866;
+    border-left-color: #226866;
+    color: #E0F5F5;
+    text-shadow: -1px -1px 0 #226866; /* same as border-bottom-color */
+}
+
+.button.helpscout:active i,
+.button.helpscout.active i {
+    filter: drop-shadow(-1px -1px 0 #226866); /* apply button's text-shadow to fontawesome */
+}
+
+/* HelpScout's default positioning wastes space and clobbers page content */
+.BeaconContainer {
+    right: 5px !important;
+    bottom: 5px !important;
```

Comment:
I hate using `!important` but for the life of me could not find another way to override HelpScout's JS-based positioning rules.